README.md

```diff
@@ -20,7 +20,7 @@
 Before you begin, you need to install the following tools:
 
 - [Node (v18 LTS)](https://nodejs.org/en/download/)
-- Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
+- [PNPM](https://pnpm.io/cli/install)
```

Comment:
I think this should point to https://pnpm.io/installation

Also maybe "pnpm" lowercase? 

lol, they are trolling in their site, when you reload it displays in a different case:

![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/2486142/20d5fb9f-b4d5-4e30-863c-74106d7cddc3)
![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/2486142/669101e3-cc85-430d-8be7-217c65c12518)

Reply:
Fixed the link here: https://github.com/scaffold-eth/scaffold-eth-2/pull/444/commits/a792ecaeb3794cac07d0d2e88446f0b9f01a375f